src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/StateSynchronizationMenuItems.cs

```diff
@@ -111,6 +111,7 @@ private static void EditCustomSettingsProperties()
                 hierarchyPrefab.AddComponent<BroadcasterSettings>();
                 hierarchyPrefab.AddComponent<SpatialLocalizationInitializationSettings>();
                 hierarchyPrefab.AddComponent<MobileRecordingSettings>();
+                hierarchyPrefab.AddComponent<NetworkConfigurationSettings>();
```

Comment:
>hierarchyPrefab.AddComponent<NetworkConfigurationSettings>(); [](start = 16, length = 61)

We might want to ensure these scripts are all present on this prefab, even if the prefab already existed. That way, as we add new functionality to SpectatorView, people who have projects already will get their prefabs updated to include the right scripts.

Reply:
This should now be updated